datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsClientFactory.java

```diff
@@ -49,4 +53,26 @@ public static Datasets getDatasetsClient() {
         }
         return _datasets;
     }
+
+    public static DataPlatforms getDataPlatformsClient() {
+        if (_dataPlatforms == null) {
+            synchronized (GmsClientFactory.class) {
+                if (_dataPlatforms == null) {
+                    _dataPlatforms = new DataPlatforms(REST_CLIENT);
+                }
+            }
+        }
+        return _dataPlatforms;
```

Comment:
what is it about the data platforms entity that means we need to have a whole fetcher for it, but something like institutional memory doesn't have a fetcher?

Reply:
From a modeling Pov, Institutional Memory is an aspect of Dataset (Eg. nested field), not a top-level entity itself. It's will not appear outside of the scope of a parent Dataset. 

Beyond that, the default `Dataset` get / batchGet methods return the full InstitutionalMemory object when retrieving a Dataset by URN, so there's no need to make another downstream call!

Reply:
Gotcha- thanks!